package.json

```diff
@@ -19,23 +29,26 @@
     "formik": "^2.4.6",
     "leaflet": "^1.9.4",
     "lexical": "^0.23.1",
-    "react": "^19.0.0-rc.1",
+    "react": "^19.0.0",
     "react-chartjs-2": "^5.3.0",
-    "react-dom": "^19.0.0-rc.1",
+    "react-dom": "^19.0.0",
     "react-draggable": "^4.4.6",
     "react-dropzone": "^14.3.5",
     "react-grid-layout": "1.4.4",
-    "react-leaflet": "^5.0.0-rc.2",
+    "react-leaflet": "^5.0.0",
     "react-resizable": "^3.0.5",
     "react-scripts": "^5.0.1",
-    "sass": "^1.83.1"
+    "sass": "^1.83.1",
+    "tslib": "^2.8.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "format": "prettier --write \"src/**/*.{ts,js,tsx,jsx,json,css,scss,html}\""
+    "format": "prettier --write \"src/**/*.{ts,js,tsx,jsx,json,css,scss,html}\"",
+    "rollup-clean-css": "sed -i '' '/@charset/d' build/dist/index.css",
+    "rollup": "rollup -c && yarn run rollup-clean-css"
```

Comment:
Could you add a publish command here to first build with rollup then publish library?